test/widgets/subscription_list_test.dart

```diff
@@ -190,4 +190,41 @@ void main() {
     check(tester.widget<UnreadCountBadge>(find.byType(UnreadCountBadge)).backgroundColor)
       .equals(swatch);
   });
+
+  testWidgets('muted streams are displayed as faded', (tester) async {
+    final stream = eg.stream(name: 'Channel 1');
+    final stream2 = eg.stream(streamId: 2, name: 'Channel 2');
+    final unreadMsgs = eg.unreadMsgs(streams: [
+      UnreadStreamSnapshot(streamId: stream.streamId, topic: 'a', unreadMessageIds: [1, 2]),
+      UnreadStreamSnapshot(streamId: stream2.streamId, topic: 'b', unreadMessageIds: [3]),
+    ]);
+    await setupStreamListPage(tester,
+      subscriptions: [
+        eg.subscription(stream, isMuted: true),
+        eg.subscription(stream2, isMuted: false)
+      ],
+      userTopics: [
+        UserTopicItem(
+          streamId: stream.streamId,
+          topicName: 'a',
+          lastUpdated: 1234567890,
+          visibilityPolicy: UserTopicVisibilityPolicy.unmuted,
+        ),
+        UserTopicItem(
+          streamId: stream2.streamId,
+          topicName: 'b',
+          lastUpdated: 1234567890,
+          visibilityPolicy: UserTopicVisibilityPolicy.unmuted,
+        ),
+      ],
+      unreadMsgs: unreadMsgs);
+
+    final channel1Finder = find.text('Channel 1');
```

Comment:
```suggestion
    final stream1Finder = find.text(stream1.name);
```
I think it would be good to stick with the name "stream" instead of "channel" for this PR as all of the codebase still uses the "stream" terminology till this point in time. This may be changed to "channel" through #631.
Also using `stream1.name` is less error-prone and requires less refactoring if the name of the stream is changed.